File: Scripts/modules/utils/paths.py
from pathlib import Path


ROOT = Path(__file__).resolve().parent.parent.parent.parent

PATHS = {
    "root": ROOT,
    "assets": ROOT / 'Scripts' / "assets",
    "mp3Folder": ROOT / 'Files' / 'MP3_files',
    "lrcFolder": ROOT / 'Files' / 'LRC_files',
    "documents": ROOT / 'Files' / 'Documents',
    "modules": ROOT / 'Scripts' / 'modules',
}

File: Scripts/utils/paths.js
import { fileURLToPath } from 'url';
import path from 'path';


const __filename = fileURLToPath(import.meta.url);
const __dirname = path.dirname(__filename);


const PROJECT_ROOT = path.resolve(__dirname, '../../');

export const PATHS = {
    root: PROJECT_ROOT,
    assets: path.join(PROJECT_ROOT, 'Scripts','assets'),
    mp3Folder: path.join(PROJECT_ROOT, 'Files', 'MP3_files'),
    lrcFolder: path.join(PROJECT_ROOT, 'Files', 'LRC_files'),
    documents: path.join(PROJECT_ROOT, 'Files', 'Documents'),
    modules: path.join(PROJECT_ROOT,'Scripts' ,'modules'),
};
